### PR TITLE
DR2-1690 Move Dynamo Formatters

### PR DESCRIPTION
**Pull request by MancunianSam**
This is now another module in the project.

### CODE REVIEW

File: ingest-upsert-archive-folders/src/test/scala/uk/gov/nationalarchives/ingestupsertarchivefolders/testUtils/ExternalServicesTestUtils.scala [hunk 1]
```diff
@@ -31,6 +32,8 @@ import scala.jdk.CollectionConverters.*
 
 class ExternalServicesTestUtils extends AnyFlatSpec with BeforeAndAfterEach with BeforeAndAfterAll with TableDrivenPropertyChecks with MockitoSugar {
 
+  case class TestIdentifier(name: String, value: String)
```
Comment (techncl) [inline ingest-upsert-archive-folders/src/test/scala/uk/gov/nationalarchives/ingestupsertarchivefolders/testUtils/ExternalServicesTestUtils.scala:35]: How come we added this?

**Review by techncl — COMMENTED** [review]
Looks good! Just 1 question, as well as thes:, 

1. Are `DynamoIdentifier` and `PreservicaIdentifier` the same thing i.e. does one eventually become the other? if so, could we just have one in the Preservica client so that we can ensure that they are consistent?
2. Do you think that maybe `EntityIdentifier` might be a clearer name than `PreservicaIdentifier`? Or maybe in the Preservica client, we could rename `Identifier` to `EntityIdentifier`?

File: ingest-upsert-archive-folders/src/test/scala/uk/gov/nationalarchives/ingestupsertarchivefolders/testUtils/ExternalServicesTestUtils.scala (hunk 1 above)
Comment (MancunianSam) [reply ingest-upsert-archive-folders/src/test/scala/uk/gov/nationalarchives/ingestupsertarchivefolders/testUtils/ExternalServicesTestUtils.scala:35]: The `Table` we're using for the tests has a mix of Preservica `Identifier` and Dynamo `Identifier`. This was fine when they were the same thing but now it's got a bit messy. I couldn't get it to compile in any kind of neat way so I introduced this as a sort of "bridge" between the two for testing purposes.

**Review by techncl — APPROVED** [review]
👍

**Pull request merged**